### PR TITLE
OSSM-3988: [DOC] Update cert-manager procedure intro for installation and verification

### PR DESCRIPTION
**Pull request by gwynnemonahan**
[OSSM-3988](https://issues.redhat.com//browse/OSSM-3988): [DOC] Update cert-manager procedure intro for installation and verification

Version(s):
4.10+

Issue:
(https://issues.redhat.com/browse/OSSM-3988)

Link to docs preview:
https://61528--docspreview.netlify.app/openshift-enterprise/latest/service_mesh/v2x/ossm-security.html#ossm-cert-manager-installation_ossm-security

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by jewertow — APPROVED** [review]

**Review by pbajjuri20 — APPROVED** [review]

File: modules/ossm-cert-manager-installation.adoc [hunk 1]
```diff
@@ -6,7 +6,7 @@
 [id="ossm-cert-manager-installation_{context}"]
 = Installing cert-manager
 
-To install cert-manager, follow these steps:
+You can install `cert-manager` to manage the lifecycle of TLS certificates; ensuring they are valid and up-to-date. If you are running Istio in your environment, you can also install `istio-csr`, a certificate authority (CA) server that handles certificate signing requests (CSR) from Istio proxies. The `istio-csr` CA delegates signing to cert-manager and cert-manager delegates to the configured CA.
```
Comment (abrennan89) [inline modules/ossm-cert-manager-installation.adoc:9]: Per https://61528--docspreview.netlify.app/openshift-enterprise/latest/service_mesh/v2x/ossm-security.html#ossm-cert-manager-integration-istio_ossm-security I'd make this consistent and instead go with "cert-manager tool" or "`cert-manager` tool" throughout the docs

Comment (abrennan89) [inline modules/ossm-cert-manager-installation.adoc:9]:
```suggestion
You can install the `cert-manager` tool to manage the lifecycle of TLS certificates and ensure that they are valid and up-to-date. If you are running Istio in your environment, you can also install the `istio-csr` certificate authority (CA) server, which handles certificate signing requests (CSR) from Istio proxies. The `istio-csr` CA delegates signing to the `cert-manager` tool, which delegates to the configured CA.
```

suggestions for clarity and minimalism

File: modules/ossm-cert-manager-installation.adoc [hunk 2]
```diff
@@ -195,7 +195,7 @@ spec:
 
 .Verification
 
-To verify cert-manager is installed, follow these steps:
+Use the sample `httpbin` service and `sleep` apps to check mTLS traffic from ingress gateways and verify that cert-manager is installed.
```
Comment (abrennan89) [inline modules/ossm-cert-manager-installation.adoc:198]:
```suggestion
Use the sample `httpbin` service and `sleep` app to check mTLS traffic from ingress gateways and verify that the cert-manager tool is installed.
```
again, be consistent with cert-manager formatting

**Review by abrennan89 — APPROVED** [review]

**Pull request merged**